socket.template.ini

```diff
@@ -67,13 +67,11 @@ file_limit = 1024
 lang = "en-us"
 maintainer = "Playmint"
 title = "PlayerchainDemo"
-<% if (process.env['VERSION']) { %>
+<% if (process.env['SS_VERSION']) { %>
 ; NOTE: Socket doesn't properly support semver versioning, it will fail to build if there are any alpha numeric chars
-version = "<%= process.env['VERSION'].split('-')[0].replace('v','') %>"
-title = "PlayerchainDemo v:<%= process.env['VERSION'] %>"
+version = "<%= process.env['SS_VERSION'].replace(/[^0-9]+/g,'.').replace(/^[.]|[.]$/g,'').split('.').slice(0,3).join('.') %>"
```

Comment:
in hindsight I didn't really need to change that split/replace ... I was just passing in things wrong while testing, so this doesn't do anything new